examples/gno.land/r/demo/mail/mail.gno

```diff
@@ -0,0 +1,37 @@
+package mail
+
+import "time"
+import "std"
+import "strings"
+
+type Mail struct {
+	id        int
+	topic     string
+	body      string
+	time      time.Time
+	sender    std.Address
+	recipient std.Address
+}
+
+func newMail(topic, body string, sender std.Address, recipient std.Address) *Mail {
+	return &Mail{counter + 1, topic, body, time.Now(), sender, recipient}
+}
+
+func (mail *Mail) Satisfies(preds []Predicate) bool {
+	for _, pred := range preds {
+		if !pred.Satisfy(mail) {
+			return false
+		}
+	}
+	return true
+}
+
+func (mail *Mail) Contains(s string) bool {
+	s = strings.ToLower(s)
+	return strings.Contains(strings.ToLower(mail.topic), s) || strings.Contains(strings.ToLower(mail.body), s)
+}
+
+// time key for avl, like 1234567890_<counter>
+func (mail *Mail) GetTimeKey() string {
+	return string(time.Now().Unix()) + "_" + string(counter)
```

Comment:
`time.Now().Unix()` and `counter ` is of type `int`. I don't think you can convert `int` to `string` by calling `string(int)`. 

Either use:
```go
        return strconv.Itoa(time.Now().Unix()) + "_" + strconv.Itoa(counter)
```
or 
```go
        return ufmt.Sprintf("%d_%d", time.Now().Unix(), counter)
```